sdk/storage/azure-storage-blob/tests/_shared/asynctestcase.py

```diff
@@ -27,7 +28,38 @@ def __init__(self):
         return self.token
 
 
+def patch_play_responses(unit_test):
+    """Fixes a bug affecting blob tests by applying https://github.com/kevin1024/vcrpy/pull/511 to vcrpy 3.0.0"""
+
+    try:
+        from vcr.stubs.aiohttp_stubs import _serialize_headers, build_response, Request, URL
+    except ImportError:
+        # return a do-nothing patch when importing from vcr fails
+        return lambda: None
+
+    def fixed_play_responses(cassette, vcr_request):
+        history = []
+        vcr_response = cassette.play_response(vcr_request)
+        response = build_response(vcr_request, vcr_response, history)
+        while 300 <= response.status <= 399:
+            if "location" not in response.headers:
+                break
+            next_url = URL(response.url).with_path(response.headers["location"])
```

Comment:
Not a comment, just a note to the reference of this:
https://github.com/kevin1024/vcrpy/blob/accffa8ea2a2d1c9c89f840b67171b9046b6fe11/vcr/stubs/aiohttp_stubs/__init__.py#L94-L108